config/jacoco.gradle

```diff
@@ -29,39 +22,27 @@ task jacocoTestReport(type: JacocoReport) {
                          'android/**/*.*']
 
     def packagesExcluded = ['com/nimbl3/di/**',
-                            'com/nimbl3/ui/**/di/**',
                             'com/nimbl3/extension/**',
                             'com/nimbl3/**/extension/**']
     // def filesExcluded = [ ]
     // def fileFilter = fileGenerated + packagesExcluded + filesExcluded
```

Comment:
redundant comments, or we could enable them back to keep a common config
```
def filesExcluded = [ ]
def fileFilter = fileGenerated + packagesExcluded + filesExcluded
```

Reply:
I have to agree, it doesn't feel like this comment has any value...

What do you mean with `could enable them back to keep a common config` ?

Reply:
Anyhow, we can keep these changes when bringing back #41 

Please keep this in mind @Wadeewee 🙏